lib/library.rb

```diff
@@ -0,0 +1,63 @@
+require 'date'
+require 'yaml'
+
+class Library
+
+  RETURN_DURATION = 1   # Value is in month
+
+  attr_accessor :books, :borrowed_book
+
+  def initialize
+
+    @books = YAML.load_file('./lib/data.yml')
+
+  end
+
+  def books_instock
```

Comment:
`books_available` or `available_books` sound like better names for this method and they're also in line with the keyword we're using to mark availability in our data file `:available` and not `:instock`